frontend/packages/console-plugin-sdk/src/typings/dashboards.ts

```diff
@@ -76,6 +76,28 @@ namespace ExtensionProperties {
     popupTitle?: string;
   }
 
+  export interface DashboardsOverviewHealthCRDSubsystem extends DashboardsOverviewHealthSubsystem {
```

Comment:
I'd call this `DashboardsOverviewHealthResourceSubsystem` as we can fetch any resource, not just CRD (although it will probably be majority of cases)

```suggestion
  export interface DashboardsOverviewHealthResourceSubsystem extends DashboardsOverviewHealthSubsystem {
```

same for all other occurrences of CRD -> Resource

Reply:
ack